dap_prinz_green_jobs/analysis/ojo_analysis/process_ojo_green_measures.py

```diff
@@ -564,6 +564,10 @@ def create_agg_data(
     """
     Much like create_agg_occ_measures but more generic to aggregate by any column
     """
+    prop_job_ads_per_itl2_all = (
+        all_green_measures_df["itl_2_name"].value_counts(normalize=True).to_dict()
```

Comment:
does normalizing do anything here that we don't want? my hunch says no but just triple checking

Reply:
no - it just gives you the proportions not the counts